server/src/controllers/auth.js

```diff
@@ -88,6 +88,9 @@ module.exports = {
 					ticket: userTicket
 				}
 			}).then((sres) => {
+				if (sres.data.response.error)
+					res.sendStatus(400).json(sres.data.response.error); // Bad request, and pass the error object as reason
```

Comment:
Hey just noting ([as from before](https://github.com/momentum-mod/website/pull/470#discussion_r585250245)) this is still needing to change to be `status(400).json(...)`.
```suggestion
					res.status(400).json(sres.data.response.error); // Bad request, and pass the error object as reason
```

Reply:
Are we still sending the error object, or should we revert this to before I changed things?

Reply:
Oh right. Then the correct thing to do here would be:
```suggestion
					res.sendStatus(400); // Bad request
```